wgsl/index.bs

```diff
@@ -7158,6 +7172,8 @@ textureSampleLevel(t : texture_depth_2d_array, s : sampler, coords : vec2<f32>,
 textureSampleLevel(t : texture_depth_2d_array, s : sampler, coords : vec2<f32>, array_index : i32, level : i32, offset : vec2<i32>) -> f32
 textureSampleLevel(t : texture_depth_cube, s : sampler, coords : vec3<f32>, level : i32) -> f32
 textureSampleLevel(t : texture_depth_cube_array, s : sampler, coords : vec3<f32>, array_index : i32, level : i32) -> f32
+textureSampleLevel(t : texture_external, s : sampler, coords : vec2<f32>) -> vec4<f32>
+textureSampleLevel(t : texture_external, s : sampler, coords : vec2<f32>, offset : vec2<i32>) -> vec4<f32>
```

Comment:
How could developer provide a valid `offset`? Restrict to 0-2?

Reply:
Offsets are restricted, below, to [-8, 7]. Should there be additional restrictions on `offset` for external textures? I'm not that familiar with sampling offsets.

They certainly would have some interesting behavior for e.g. [NV12](https://www.fourcc.org/pixel-format/yuv-nv12/) where the UV plane is quarter-resolution.
`textureSampleLevel(t, s, coords, offset)` would I guess would translate to something like
`colorConvert(t, textureSampleLevel(t.yPlane, s, transformedCoords, 0, transformedOffset), textureSampleLevel(t.uvPlane, s, transformedCoords, 0, floor(transformedOffset / 2)))`?
I'm not sure this is the behavior they would expect from `offset`.
Maybe it should be omitted.

Reply:
Chatted about this at office hours. That translation is actually impossible because transformedOffset can't be computed statically, but offsets must be static. So we would have to fold the offset together into the sample coords. In which case it provides no functionality the user couldn't do themselves. Removing this until there's a real identified use case (I don't think there will be one).

Reply:
I think the deal-breaker for const-offset is incoming videos that have rotations, where offset directions will go in unexpected directions. I think we should drop `offset` for texture_external